java/compaction/compaction-status-store/src/test/java/sleeper/compaction/status/store/job/StoreCompactionJobUpdatesIT.java

```diff
@@ -58,6 +58,26 @@ public void shouldReportCompactionJobStartedSeparatelyFromCreation() {
                 .containsExactly(startedStatusWithDefaults(job));
     }
 
+    @Test
+    public void shouldReportCompactionJobUncommittedSeparatelyFromOthers() {
```

Comment:
This test seems to be an exact duplicate of shouldReportCompactionJobUncommitted.